### DISCUSSION TITLE
&amp;amp;

### DISCUSSION THREAD

**Discussion by lildiablo** [maizzle/framework#1368] [General]
All my special character are turninng in to something like:
```&amp;amp;
&lt and &gt as well 
also <br /> was changing to <br></br>

**Comment by cossssmin** [comment]
Which version are you using? Can you please provide a reproduction?

**Reply by lildiablo** [reply]
"@maizzle/framework": "latest",

```
<p class="font-arial m-0 mb-6 text-base text-black">
            Dear &lt;insert name&gt
          </p>
```

```
<p style="margin: 0 0 24px; font-family: Arial, Helvetica, sans-serif; font-size: 16px; color: #000001">
              Dear &amp;lt;insert name&amp;gt
            </p>
            ```

**Reply by lildiablo** [reply]
also i've found that outlook "sometime" is weird with margin: x x x x; iinstead of magint-top: x; margin-bottom: x; any thoughts?

**Reply by cossssmin** [reply]
Sorry for not being explicit, was thinking of a repo that reproduces the issue (that includes `package-lock.json`) - this is what I get with a fresh 4.x (latest) project so there might be something else causing it:

![image](https://github.com/user-attachments/assets/d4ba001d-20ff-4f57-85a3-59a5b0bcec85)

For `<br />` being turned into `<br>`, that is expected as the parser is not running in XML mode. You can change that in the [PostHTML config](https://maizzle.com/docs/configuration/posthtml#xmlmode).

**Reply by cossssmin** [reply]
> also i've found that outlook "sometime" is weird with margin: x x x x; iinstead of magint-top: x; margin-bottom: x; any thoughts?

Don't remember Outlook not supporting shorthand margins, only things I can think of is if using unsupported units or trying to use `margin` on inline elements.

**Reply by cossssmin** [reply]
Actually for `<br />` I think this would be better:

https://maizzle.com/docs/configuration/posthtml#singletags

Set that to `slash` for `<br>`:

```js
module.exports = {
  build: {
    posthtml: {
      options: {
        singleTags: ['br'],
        closingSingleTag: 'slash'
      }
    },
  },
}
```

**Comment by lildiablo** [comment]
what's your bitbucket handle?

**Reply by cossssmin** [reply]
`cosmin-popovici`

**Comment by lildiablo** [comment]
sorry it's saying "Invite people to the workspace
Add people by email"
please mail me at 
helldiablo [at] gmail

**Reply by cossssmin** [reply]
think it would be easier to just create a private repo here instead?
